src/api/lib/tasks/errbit.rake

```diff
@@ -0,0 +1,6 @@
+namespace :errbit do
+  desc 'Raise an exception, can be used to test erbit setups'
+  task :test do
+    raise RuntimeError.new("Runtime error exception to test error handling")
```

Comment:
I dont think we do, I sent out that email to the mailing list and nobody said anything so I assume we dont. If we need to test errbit in the future we can just SSH in and run this rake task.